install_config/build_defaults_overrides.adoc

```diff
@@ -28,19 +28,25 @@ allows administrators to change the global configuration at any time, and any
 builds that are re-run from existing build configurations or builds will get the
 new settings.
 
-The `BuildDefaults` admission control plug-in allows administrators to set
+* The `BuildDefaults` admission control plug-in allows administrators to set
 global defaults for settings such as the Git HTTP and HTTPS proxy, as well as
 default environment variables and image labels. These defaults do not overwrite
 values that have been configured for a specific build. However, if those values
 are not present on the build definition, they are set to the default value.
 
-The `BuildOverrides` admission control plug-in allows administrators to
+* The `BuildOverrides` admission control plug-in allows administrators to
 override a setting in a build, regardless of the value stored in the build. It
 currently supports overriding the `forcePull` flag on a build strategy to
 enforce always refreshing the local image during a build by pulling the image
 from the registry. This ensures that a user can only build with an image that
 they are allowed to pull. The plug-in can also be configured to apply a set of
 image labels to every built image.
++
+[NOTE]
+====
+You can define a hard limit on how long build pods can run by using the 
+xref:../admin_guide/managing_pods.adoc#manage-pods-limit-run-once-duration[RunOnceDuration plugin].
```

Comment:
`RunOnceDuration` plug-in

Reply:
@ahardin-rh The section I linked to uses *RunOnceDuration*. So, I went with that for consistency/